benchmark/Microsoft.IdentityModel.Benchmarks/AntiVirusFriendlyConfig.cs

```diff
@@ -1,18 +1,97 @@
+        public bool IsNumeric => true;
+        public UnitType UnitType => UnitType.Time;
+        public string Legend => "50th percentile of latency";
+    }
+
+    public class P99Column : IColumn
+    {
+        public string Id => nameof(P99Column);
+        public string ColumnName => "P99";
+        public bool IsDefault(Summary summary, BenchmarkCase benchmarkCase) => false;
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase) => GetValue(summary, benchmarkCase, SummaryStyle.Default);
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase, SummaryStyle style)
+        {
+            // Get the statistics for the current benchmark
+            var statistics = summary[benchmarkCase].ResultStatistics;
+            // Calculate the P99 latency using the Percentile method
+            var p99 = statistics.Percentiles.P90 / style.TimeUnit.NanosecondAmount;
+            // Format the value using the style
+            return p99.ToString($"F2") + $" {style.TimeUnit.Name}";
+        }
+        public bool IsAvailable(Summary summary) => true;
+        public bool AlwaysShow => true;
+        public ColumnCategory Category => ColumnCategory.Statistics;
+        public int PriorityInCategory => 0;
+        public bool IsNumeric => true;
+        public UnitType UnitType => UnitType.Time;
+        public string Legend => "99th percentile of latency";
+    }
+
+    public class P100Column : IColumn
+    {
+        public string Id => nameof(P100Column);
+        public string ColumnName => "P100";
+        public bool IsDefault(Summary summary, BenchmarkCase benchmarkCase) => false;
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase) => GetValue(summary, benchmarkCase, SummaryStyle.Default);
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase, SummaryStyle style)
+        {
+            // Get the statistics for the current benchmark
+            var statistics = summary[benchmarkCase].ResultStatistics;
+            // Calculate the P99 latency using the Percentile method
+            var p99 = statistics.Percentiles.P100 / style.TimeUnit.NanosecondAmount;
+            // Format the value using the style
+            return p99.ToString($"F2")+ $" {style.TimeUnit.Name}";
+        }
+        public bool IsAvailable(Summary summary) => true;
+        public bool AlwaysShow => true;
+        public ColumnCategory Category => ColumnCategory.Statistics;
+        public int PriorityInCategory => 0;
+        public bool IsNumeric => true;
+        public UnitType UnitType => UnitType.Time;
+        public string Legend => "100th percentile of latency";
+    }
+
+
     public class AntiVirusFriendlyConfig : ManualConfig
     {
         public AntiVirusFriendlyConfig()
         {
             AddJob(Job.MediumRun
                 .WithToolchain(InProcessEmitToolchain.Instance));
+            AddColumn(new P50Column(), new P99Column(), new P100Column());
```

Comment:
I think there might be some value in adding a P95, It gives us a little more idea of what the tail values look like.

Reply:
I've added P95 as well. Thanks @keegan-caruso